src/ashtech.h

```diff
@@ -72,16 +72,16 @@ class GPSDriverAshtech : public GPSHelper
 		NME_DECODE_GOT_FIRST_CS_BYTE
 	};
 
-	struct satellite_info_s *_satellite_info;
-	struct vehicle_gps_position_s *_gps_position;
-	uint64_t _last_timestamp_time;
-	int _ashtechlog_fd;
+	struct satellite_info_s *_satellite_info {nullptr};
```

Comment:
Yes, that would have initially been my choice as well. 
It seems to be the preference in c++ core guidelines. It's definitely what the tools want to do by default. Let me see if it actually says anything.